src/processors/default/processMempool.ts

```diff
@@ -0,0 +1,64 @@
+
+
+import _ from 'lodash';
+
+import { Table } from '../../db/db';
+import { pendingMempoolMessages } from '../../triggers/messages';
+import { CrdtOpetation, CrdtState, MempoolMessage, PendingMempoolMessage } from '../../types/message';
+import { Clients, Processor } from '../../types/processor';
+
+export const processMempool: (table: Table) => Processor = 
+  (table) => ({
+
+    name: 'processMempool',
+    trigger: pendingMempoolMessages(table),
+    processorFunction: async (messages: PendingMempoolMessage[] , clients: Clients) => {
+      // ignore inserts for now
+      const staleMessages: MempoolMessage[] = messages.filter(message => message.operation === CrdtOpetation.INSERT);
+
+      // messages are ordered by timestamp.
+      //  group by table, column, entity so that we can easily categorize fresh and stale messages
```

Comment:
```suggestion
      //  group by table, column, entity so that we can easily categorize fresh and stale messages
      // messages within each group are ordered by timestamp in the trigger, so it's safe to assume the first message of each group is the freshest
```